examples/llms/providers/customChatProvider.ts

```diff
@@ -77,6 +79,10 @@ export class CustomChatLLM extends ChatLLM<CustomChatLLMOutput, CustomGenerateOp
     return { tokenLimit: Infinity };
   }
 
+  async embed(_input: BaseMessage[][], _options?: EmbeddingOptions): Promise<EmbeddingOutput> {
+    throw new Error("Method not implemented.");
```

Comment:
Use the framework's built-in `NotImplementedError` class.

Reply:
This has been resolved by @vabarbosa